app/build.gradle

```diff
@@ -29,13 +29,7 @@ dependencies {
     implementation "org.pf4j:pf4j:${pf4jVersion}"
     implementation "org.apache.commons:commons-lang3:${commonsLangVersion}"
     implementation "org.yaml:snakeyaml:${snakeyamlVersion}"
-    implementation "com.github.kittinunf.fuel:fuel:${fuelVersion}"
-    implementation "com.github.kittinunf.fuel:fuel-json:${fuelVersion}"
-    constraints {
-        implementation("org.json:json:${jsonVersion}") {
-            because("Required until fuel-json updates it's internal version of org.json, not fixed as of fuel version 2.3.1")
-        }
-    }
+    implementation "org.json:json:${jsonVersion}"
```

Comment:
What happens if we delete:
```
implementation "org.json:json:${jsonVersion}"
```?

Also, I think we can delete `fuelVersion` Gradle constant.

Reply:
Looks like it is ok without that dependency